release-notes/cabal-install-3.12.1.0.md

```diff
@@ -1,12 +1,16 @@
-Pre-release cabal-install 3.12.0.0/3.11.0.0 changelog and release notes.
-
-This file will be edited and the changes incorprated into the official
-3.12.1.0 cabal-install and cabal-install-solver release notes.
-
+cabal-install 3.12.1.0 changelog and release notes.
 ---
 
 ### Significant changes
 
+- Cabal 3.12 support [#9917](https://github.com/haskell/cabal/issues/9917)
+
+    This is the first release of `cabal-install` that is fully compatible with `Cabal` 3.12.0.0 as released with GHC 9.10.1. In particular, it means custom setup builds will work with GHC 9.10.
+
+- `cabal-install` has been built against `tar` 0.6.3.0 [#10123](https://github.com/haskell/cabal/pull/10123)
```

Comment:
This is the binary only, we just bumped the index, should we mention this?

Reply:
It was suggested to mention it due to the significant speedups resulting from the new version.

Reply:
Well, binaries are a part of release. I think no one is terribly interested in our index-state. I agree though that the notes can say something to the users of our source part of the release rather than the binary part of it. E.g. it could say:

> For anyone building from the source, we suggest making sure that the version of `tar` in your plan is at least 0.6.3.0. (For example, we bumped the index-state in our cabal.project.release in the Cabal repository.)

Or something along those lines...

Reply:
Yeah that is what I meant!